src/utilities/portals/PortalsContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {useContext} from 'react';
+
+import {PortalsManagerContext} from './context';
+
+export function PortalsContainer() {
+  const portalsManagerContext = useContext(PortalsManagerContext);
+
+  if (!portalsManagerContext) {
+    return null;
+  }
+
+  const {setContainerNode} = portalsManagerContext;
+
+  return <div id="PolarisPortalsContainer" ref={setContainerNode} />;
```

Comment:
Should this always be rendered @alex-page?

I'm working on updating from Polaris 5.8.4 to 5.10.1 https://github.com/Shopify/online-store-web/pull/8774

But seeing [this error on buildkite](https://buildkite.com/shopify/online-store-web-ci-builder/builds/37908#4275f8f1-ace9-4d88-9a86-2fc3da26d70d) for one of our tests 
<img width="695" alt="image" src="https://user-images.githubusercontent.com/7477411/99268066-b3614f80-27f2-11eb-89b4-b5d976b4f290.png">

The component should be returning `null` if not rendered. But with this Polaris update, Polaris seems to always insert this `div` into the DOM. 

Component being tested https://github.com/Shopify/online-store-web/blob/master/app/sections/Themes/ThemeIndex/ThemeIndex.tsx#L324

Failing test https://github.com/Shopify/online-store-web/blob/master/app/sections/Themes/ThemeIndex/tests/ThemeIndex.test.tsx#L107

Reply:
Hey @MLDimitry. Yep, you are correct. The way the portals work now this container is always rendered. Your test makes the assumption that none of the providers also output markup (not just your component). I would like to make this more automatic but for now you could update your test to something like `expect(wrapper.find(ThemeIndex).html()).toBe('');` to be a bit more specific.

Reply:
Thanks for confirming what I figured + the context, I'll update the test to be more specific then 👍